stress_test_model_eq_cb.R

```diff
@@ -409,7 +412,10 @@ scenario_data <- scenario_data %>%
       scenario_geography == scenario_geography_filter)
 
 # Load price data----------------------------------------
-df_price <- readr::read_csv(file.path(data_location, paste0("prices_data_", price_data_version, ".csv")), col_types = "ncccccncncncnc") %>%
+df_price <- read_price_data(
+    path = file.path(data_location, paste0("prices_data_", price_data_version, ".csv")),
+    version = "old"
```

Comment:
I saw that version is always old though price_data_version differs between the 3 calls of the function. Is that intentional?

Reply:
yes, only price_data_version == 2021Q3 would be version new as of now. It is currently not being used anywhere